index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="icon" href="images/logo/favicon.ico" />
+
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+
+  <body>
+    <header>
+      <a href="/"
+        ><img
```

Comment:
💊 제안
현재 코드에서는 한 태그가 두 줄로 나눠 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
      <a href="/">
        <img
```